pkg/system/resources.go

```diff
@@ -1,76 +1,73 @@
 package system
 
 import (
-	"os"
+	"fmt"
+	"strings"
 
 	corev1 "k8s.io/api/core/v1"
+
 	"k8s.io/apimachinery/pkg/api/resource"
 )
 
-// Values will likely need to be tweaked as we get more usage data. They are currently based
-// on metrics we have collected from internal use. You can override these values by setting
-// the corresponding environment variable.
-var (
-	mi = int64(1 << 20) // 1 MiB in bytes
-	gi = int64(1 << 30) // 1 GiB in bytes.
-
-	registryMemoryRequest = *resource.NewQuantity(128*mi, resource.BinarySI)    // REGISTRY_MEMORY_REQUEST
-	registryMemoryLimit   = *resource.NewQuantity(512*mi, resource.BinarySI)    // REGISTRY_MEMORY_LIMIT
-	registryCPURequest    = *resource.NewMilliQuantity(200, resource.DecimalSI) // REGISTRY_CPU_REQUEST
+var ErrInvalidResourceSpecification = fmt.Errorf("invalid resource specification")
 
-	buildkitdMemoryRequest = *resource.NewQuantity(256*mi, resource.BinarySI)    // BUILDKITD_MEMORY_REQUEST
-	buildkitdMemoryLimit   = *resource.NewQuantity(10*gi, resource.BinarySI)     // BUILDKITD_MEMORY_LIMIT
-	buildkitdCPURequest    = *resource.NewMilliQuantity(800, resource.DecimalSI) // BUILDKITD_CPU_REQUEST
+func ValidateResources(resources ...string) error {
+	// TODO - fix
```

Comment:
Does something still need to be changed or was this left behind?

Reply:
This was left behind.